packages/ui-react/src/components/LayoutGrid/LayoutGrid.tsx

```diff
@@ -0,0 +1,153 @@
+// Keyboard-accessible grid layout, with focus management
+
+import { throttle } from '@jwp/ott-common/src/utils/common';
+import useEventCallback from '@jwp/ott-hooks-react/src/useEventCallback';
+import { useEffect, useLayoutEffect, useRef, useState } from 'react';
+
+import styles from './LayoutGrid.module.scss';
+
+type Props<Item> = {
+  className?: string;
+  columnCount: number;
+  data: Item[];
+  renderCell: (item: Item, tabIndex: number) => JSX.Element;
+};
+
+const scrollIntoViewThrottled = throttle(function (focusedElement: HTMLElement) {
```

Comment:
When I click elements close to the bottom of the page I also get auto-scrol and the link itself doesn't work.

https://github.com/jwplayer/ott-web-app/assets/27254986/cc5a1301-8fb4-4ae7-81d4-4f544ee0a6a7

Reply:
This is fixed